tests/game.test.js

```diff
@@ -0,0 +1,109 @@
+const each = require('jest-each').default;
+const { Game } = require('../server/game');
+
+describe('hasTurnReset', () => {
```

Comment:
The name of the test in the describe call would normally directly refer to the unit under test. For example, this test tests the `resetGame` method:
```suggestion
describe('resetGame', () => {
```